cs50/2017/ap/calc/check50/__init__.py

```diff
@@ -0,0 +1,59 @@
+from check50 import *
+
+
+class Calc(Checks):
+
+    @check()
+    def exists(self):
+        """calc.c exists"""
+        self.require("calc.c")
+
+    @check("exists")
+    def compiles(self):
+        """calc.c compiles"""
+        self.spawn("clang -o calc calc.c -lcs50 -lm").exit(0)
+
+    @check("compiles")
+    def test_handles_addition(self):
+        """calculator handles addition"""
+        self.spawn("./calc 3 + 4").stdout("7.000000\n").exit(0)
```

Comment:
This will accept any number ending with `7.000000` e.g. `47.000000`. You could use a regex like the `number` one you use in Fahrenheit.